### PR TITLE
fixes #84

### LINKED ISSUE #84: Users should be able to close embeded photos or videos
Users should be able to close embedded photos or videos, like with irccloud where embedded/inline content can be dismissed if you don't want it to take up space in the chat. Or maybe minimize embedded content.

### PR DESCRIPTION
**Pull request by geoffwhittington**

### CODE REVIEW

File: media/js/views/room.js [hunk 1]
```diff
@@ -390,6 +392,15 @@
...
             Backbone.View.prototype.remove.call(this);
         },
+        deleteObject: function(e) {
+            var target = $(e.currentTarget),
+                objectId = target.data('id'),
+                object = this.$('#' + objectId);
+
+            if (!object) return;
+
+            object.remove();
```
Comment (geoffwhittington) [inline media/js/views/room.js]: Maybe we want .toggle() here instead?

**Comment by geoffwhittington** [comment]
We need to replace the image with it's link

**Comment by hhaidar** [comment]
This should be a menu option I think

**Comment by geoffwhittington** [comment]
Care to expand what you mean by menu option - inline all images or not?

**Comment by hhaidar** [comment]
@geoffwhittington yea something like "Enable thumbnails for images" and it would just toggle all of them on/off at will.

**Comment by geoffwhittington** [comment]
What do you think of adding a thing people could click to selectively inline a specific image?

**Comment by sibartlett** [comment]
A menu option might be nice, but I'd like to be able to selectively close images too.

**Comment by geoffwhittington** [comment]
@sibartlett Sounds like you want some sort of toggle

**Comment by geoffwhittington** [comment]
Punted

**Pull request closed without merging**